src/Jackett.Common/Indexers/TorrentDay.cs

```diff
@@ -49,7 +49,7 @@ private new ConfigurationDataRecaptchaLogin configData
         public TorrentDay(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
             : base(name: "TorrentDay",
                 description: "TorrentDay (TD) is a Private site for TV / MOVIES / GENERAL",
-                link: "https://torrentday.com/",
+                link: "https://www.torrentday.com/",
```

Comment:
1. please add `https://torrentday.com/` to the `LegacySiteLinks` section, so that users whose config currently point to ``https://torrentday.com/`` will have their `SiteLink` automatically changed to `https://www.torrentday.com/`
2. please also delete the duplicate `https://www.torrentday.com/` in the `AlternativeSiteLinks` section, it's in there twice for some obscure reason
3. and while you're there, delete the `https://torrentday.it/` as well, since this is no longer working. ;-)
Thanks.

Reply:
There we go, suggested changes should have been applied now.